ui/pages/snap-account-redirect/components/url-display-box.tsx

```diff
@@ -1,5 +1,5 @@
 import React from 'react';
-import { SnapAccountRedirectProps } from '../snap-account-redirect';
+import { type SnapAccountRedirectProps } from '../snap-account-redirect';
```

Comment:
should be this?
```suggestion
import type { SnapAccountRedirectProps } from '../snap-account-redirect';
```

Reply:
these are effectively the same thing in practice. I don't care either way though, so I've gone ahead and updated both instances.